src/locales/en/login.json

```diff
@@ -0,0 +1,14 @@
+{
+    "forgotPassword": "Forgot password?",
+    "formLabel": "log in",
+    "rememberMe": "Remember me",
+    "registerProposal": "Registration",
+    "placeholders": {       
+        "email": "Email",
+        "password": "Password"        
+    },
+    "orText": "or",
```

Comment:
It's a bad idea to put "or" in translation, cause not in all languages we have it. Better to use full sentences.

But if you need only "or", then create general file, where we can keep general translations, like "or" "email", and other common words or sentences.